### PR TITLE
Fixes #2691 Can create multiple usergroups with the same name

### LINKED ISSUE #2691: Can create multiple usergroups with the same name
I'd consider this a bug so I'll post it here and see if it effects anyone else. I can create many user groups with the exact same group name. There should be some sort of check.

Original thread: https://community.mybb.com/thread-207529.html

### PR DESCRIPTION
**Pull request by JordanMussi**
Fixes #2691 Can create multiple usergroups with the same name

---

**This PR:**
- changes 1 language file

### CODE REVIEW

File: admin/modules/user/groups.php [hunk 1]
```diff
@@ -743,7 +743,15 @@
 			// Log admin action
 			log_admin_action($gid, htmlspecialchars_uni($mybb->input['title']));
 
-			flash_message($lang->success_group_created, 'success');
+			$groups = $cache->read('usergroups');
+			$grouptitles = array_column($groups, 'title');
+
+			$message = $lang->success_group_created;
+			if(in_array($mybb->input['title'], $grouptitles) && count(array_keys($grouptitles, $mybb->input['title'])) > 1) {
```
Comment (Shade-) [inline admin/modules/user/groups.php]: The bracket should be in a new line

File: admin/modules/user/groups.php [hunk 2]
```diff
@@ -964,7 +972,15 @@
 			// Log admin action
 			log_admin_action($usergroup['gid'], htmlspecialchars_uni($mybb->input['title']));
 
-			flash_message($lang->success_group_updated, 'success');
+			$groups = $cache->read('usergroups');
+			$grouptitles = array_column($groups, 'title');
+
+			$message = $lang->success_group_updated;
+			if(in_array($mybb->input['title'], $grouptitles) && count(array_keys($grouptitles, $mybb->input['title'])) > 1) {
```
Comment (Shade-) [inline admin/modules/user/groups.php]: Same here

File: admin/modules/user/groups.php (hunk 1 above)
Comment (JordanMussi) [reply admin/modules/user/groups.php]: Fixed in 85e640af00f8be38b5ac5e50fcfea3d7850c5727

File: admin/modules/user/groups.php (hunk 2 above)
Comment (JordanMussi) [reply admin/modules/user/groups.php]: Fixed in 85e640af00f8be38b5ac5e50fcfea3d7850c5727

**Comment by Cu8eeeR** [comment]
Works fine, thank you!

**Pull request merged**